docs/installation/pipx-based.rst

```diff
@@ -35,7 +35,7 @@ In the following sections, we will guide you step by step in configuring your sy
 Launching a terminal
 ~~~~~~~~~~~~~~~~~~~~
 
-All operating systems have a Terminal application that let you run commands. You will need to use it extensively when using MUSE, so we strongly suggest you get familiar with it. For now, let's just figure out how to launch it:
+All operating systems have a Terminal application that let you run commands. You will need to use it extensively when using MUSE, so we strongly suggest you get familiar with it. For now, let us just figure out how to launch one:
```

Comment:
I'd leave this as is. It's a minor point, but "let us" sounds overly formal. We generally try to strike a friendly, informal tone in documentation